src/main/lws/lws-service.js

```diff
@@ -172,16 +212,18 @@ export class LWSService {
 			const body = {
 				publicKey: msg.payload.publicKey,
 				nonce: nonceResp.nonce,
-				signature: this.genSignature(nonceResp.nonce, msg.payload.publicKey, conn)
+				signature: Buffer.from(this.stringifySignature(signature), 'utf8').toString(
```

Comment:
we should include this together with the signature generation, modification functions and then just return the final result to the request body object